app/src/test/java/org/simple/clinic/summary/PatientSummaryUpdateTest.kt

```diff
@@ -377,7 +377,7 @@ class PatientSummaryUpdateTest {
     updateSpec
         .given(model)
         .whenEvent(DataForDoneClickLoaded(
-            hasPatientMeasurementDataChangedSinceScreenCreated = true,
+            hasPatientMeasurementDataChangedSinceScreenCreated = false,
```

Comment:
Shouldn't this be `true`? according to the test name? `when patient measurement data is changed`

Reply:
Huh, it should be. I am not sure how this got reset. Good catch, taking a look at it.

Reply:
OMG, this is from an entirely different test, because that test name was not changed. It seemed like it's from the same test.